src/pages/extending-agent-app/products-sdk/agent-app-sdk/index.mdx

```diff
@@ -443,6 +443,72 @@ widget.setReportsFilters({ tag: ['chatbot'] });
 
 </CodeSample>
 
+## Settings widgets
+
+If you want to connect a Settings widget to the LiveChat App, you should use the `createSettingsWidget` function. It returns a promise resolving to a Settings widget instance.
+
+<Code>
+<CodeSample path={'SETTINGS WIDGETS'}>
+
+```js
+import { createSettingsWidget } from "@livechat/agent-app-sdk";
+
+createSettingsWidget().then(widget => {
+  // build your logic around the widget
+});
+```
+
+</CodeSample>
+</Code>
+
+### Events
+
+#### `page_data`
+
+Emitted when the widget is initialized. The handler will get the main window page data object as an argument.
+
+Listen for the page data changes:
+
+<Code>
+<CodeSample path={'PAGE_DATA'}>
+
+```js
+widget.on("page_data", pageData => {
+  // read the page data
+});
+```
+
+</CodeSample>
+</Code>
+
+### Methods
+
+#### Get page data
+
+Gets the most recently recorded main window page data and returns the `IPageData` object (identical to the one emitted by the `page_data` event). If no data was recorded, it returns null.
```

Comment:
```suggestion
Gets the most recently recorded main window page data and returns the `IPageData` object (identical to the one emitted by the `page_data` event). If no data was recorded, it returns `null`.
```

Reply:
I now noticed other nulls on the page are as snippets as well, so just this one small change and I'll merge the PR 🙏

Reply:
116a26eb652fbf852616baa506c9451b12048e14